zebra-consensus/src/verify.rs

```diff
@@ -1,350 +1,10 @@
-//! Block verification and chain state updates for Zebra.
+//! Block and transaction verification for Zebra.
```

Comment:
we might want to flatten the zebra-consensus module tree at some point, with, e.g., `::block`, `::transaction`, `::checkpoint`, etc. rather than `::verify::{block, script, transaction}` since most of the work we're doing is verification. but this is something to figure out as we go along, not a requested change on this pr specifically

Reply:
I'd like to do large code movement after this PR merges.

Reply:
See #582.